Tutorials/Basic/HeatEquation_EX1_C/Source/main.cpp

```diff
@@ -90,7 +90,7 @@ void main_main ()
     init_phi(phi_new, geom);
     // ========================================
 
-    Real dt = 0.9_rt*dx[0]*dx[0] / (2.0_rt*AMREX_SPACEDIM);
+    Real dt = 0.9_rt*(dx[0]*dx[0] + dx[1]*dx[1] + dx[2]*dx[2])/ (2.0_rt*AMREX_SPACEDIM);
```

Comment:
I think it's supposed to be `dx**3/3` -> `1./((1./dx)**2+(1./dy)**2+(1./dz)**2)` if dx, dy and dz are not equal to each other.

Reply:
Is there a macro that'll clean this up? I don't believe D_TERM will work:

`#if (AMREX_SPACEDIM==1)
    Real coeff = (1./dx[0]**2);
#elif (AMREX_SPACEDIM==2)
    Real coeff = (1./dx[0]**2) + (1./dx[1]**2);
#else 
    Real coeff = (1./dx[0]**2) + (1./dx[1]**2) + (1./dx[2]**2);
#endif

    Real dt = 0.9_rt/ (2.0_rt*length);
`